### DISCUSSION TITLE
Automatic script for performance testing TMail backend

### DISCUSSION THREAD

**Discussion by quantranhong1999** [linagora/tmail-backend#772] [Ideas]
Hi guys,
WDYT about an automatic script for performance testing TMail backend and notifying the result (e.g through mail or chatbot)?

E.g either we do it as a cron job for master every day or we trigger the perf test via Jenkins upon a new commit/new tag.

If we succeed to setup this, IMO we would better track the performance of TMail as well as spend fewer resources on manually doing this (e.g on every release).

**Comment by Arsnael** [comment]
Ok but only on master branch maybe during the night. Need an efficient way to share the result as well. the maste branch should be redeployed every day before the run (cronjob)

**Comment by chibenwa** [comment]
+1 with https://github.com/linagora/james-project-private/issues/1205 nightly perf test runs seem achievable, just need to automate the deployment of testing env, and ensure it do not screw up with perf tests humanly scheduled on releases.
